README.md

```diff
@@ -1,31 +1,52 @@
-# osl-php
-
-
-Packages Recipe
-===============
-
-Using the packages recipe has a few rules. It will iterate over <code>node['osl-php']['packages']</code>
-and any Enumerable objects nested inside. For any String objects it finds, it installs them.
-
-For any Enumerable object found, there are three cases:
-
-1. 0 objects in the Enumerable object are Enumerable.
-2. At least one object in the Enumerable object is an Enumerable object,
-3. All objects in the Enumerable object are enumerable.
-
-An example for each case is the following:
-
-1. <code>h = ['php5-gd', 'php5-fpm']</code>
-2. <code>h = ['php5-gd', {'php5-fpm' => true}]</code>
-3. <code>h = [{'php5-gd' => true, 'php5-fpm' => true}]</code>
-
-The cases are solved in the following ways:
-
-1. Include every item in <code>h</code>.
-2. Include non-hash items, include all hash keys that have a true value
-3. Same as two, with no non-hash items.
-
-The end result will be a list of packages to install. For example, all 3 cases produce the following list:
-
-<code>['php5-gd', 'php5-fpm']</code>
-
+# osl-php Cookbook
+
+Attributes
+----------
+* `node['osl-php']['use_ius']` - Use PHP packages from [IUS Community](https://ius.io/) repositories when true.
+* `node['osl-php']['packages']` - PHP packages to install. (The primary package does not have to be included here.)
+* `node['osl-php']['short_packages']` - PHP packages to install specified without the "php-" or "php71u-" prefixes.
+
+Usage
+-----
+
+### osl-php::packages
+The packages recipe makes use of the three attributes described above (`use_ius`, `packages`, and `short_packages`).
+`use_ius` should be set to true or false depending on whether you're using PHP packages from IUS Community repos.
+
+The primary PHP package is installed by this recipe automatically, and does not have to be specified in attributes. `packages` and `short_packages` are for additional PHP packages.
+
+Most PHP packages have a "php" prefix in their names, like "php-memcached".
+Packages from IUS have versioned prefixes, like "php56u-memcached" or "php71u-memcached".
+Packages that have these prefixes should be added to `short_packages` without the prefix.
+Instead of adding "php-memcached" or "php71u-memcached", just add "memcached".
+The recipe will handle adding the proper prefixes, depending on `node['php']['version']` (when using IUS).
+
+Packages that don't have this prefix (e.g. "pear1u") should be placed in the `packages` attribute.
+
+#### Examples
+
+```ruby
+node.default['osl-php']['use_ius'] = false
+node.default['osl-php']['short_packages'] = %w(devel cli pear)
+node.default['osl-php']['packages'] = []
```

Comment:
This isn't necessary if no packages are specified, the attribute file sets this to an empty array by default

Reply:
Yeah, my intention was that this would make it more clear what the values of each attribute are for the given outcome.
However, I guess this could imply the attribute should be cleared for some reason. I'll take it out.